mobile/src/stateMachine/views/notificationCustomize/notificationsCustomize.tsx

```diff
@@ -0,0 +1,194 @@
+                            title="Sleep Domain"
+                            description={notificationsEnabled ? 'On' : 'Off'}
+                            style={{ marginBottom: 20 }}
+                            Image={Images.bellIcon}
+                        >
+                            <Switch
+                                value={this.model.isEnabled}
+                                disabled={this.model.isToggleInProgress}
+                                style={styles.switchStyles}
+                            />
+                        </Card>
+                        <Card
+                            title="Include notifications that  mention bipolar diagnosis"
+                            description={notificationsEnabled ? 'On' : 'Off'}
+                            style={{ marginBottom: 20, height: 100 }}
+                            Image={Images.bellIcon}
+                        >
+                            <Switch
+                                value={this.model.isEnabled}
+                                disabled={this.model.isToggleInProgress}
+                                style={styles.switchStyles}
+                            />
+                        </Card>
+                        <View style={styles.buttonView}>
+                        <Button
+                            title="SAVE"
+                            style={[styles.insturctionsButton, this.textStyles.h2]}
+                            titleStyles={styles.insturctionsButtonTitle}
+                            onPress={this.onNext}
+                            isTransparent
+                            />
+                        </View>
+                    </Container>
+                </ScrollView>
+            </MasloPage>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    topBarWrapWrap: {
+        position: 'absolute',
+        top: 0,
+        width: '100%',
+        zIndex: 2,
+        elevation: 2,
+    },
+    topBarWrap: {
+        position: 'relative',
+        flexDirection: 'row',
+        alignItems: 'center',
+        justifyContent: 'flex-start',
+        height: 72,
+        zIndex: 2,
+        elevation: 2,
+    },
+    backBtn: {
+        width: 52,
+        height: 52,
+        alignItems: 'flex-start',
+        justifyContent: 'center',
+        backgroundColor: 'transparent',
+    },
+    container: {
+        minHeight: Layout.window.height,
+        paddingTop: Layout.getViewHeight(21),
+    },
+    title: {
+        marginBottom: 40,
+        textAlign: 'center',
+    },
+    exactCard: {
+        borderLeftWidth: 20,
+        borderBottomLeftRadius: 0,
+        borderBottomRightRadius: 0,
+        borderBottomWidth: 0,
+        marginBottom: 0,
+        color: 'grey',
+    },
+    switchStyles: {
+        paddingHorizontal: 3,
+    },
+    buttonView : {
+        alignContent: 'center',
+        alignItems: 'center',
+        padding: 5
+     },
+    insturctionsButton: {
+        width: 320,
```

Comment:
Change this width to a percentage (eg, 80%) so it scales nicely on different screen sizes